### PR TITLE
test: add integration test for `pubsub` API

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**

### CODE REVIEW

File: test/cli/test_detect.py [hunk 1]
```diff
@@ -190,6 +189,47 @@ def test_oslogin_proto_beta(self):
...
+            )
+            self.assertEqual(result.exit_code, 0)
+            self.assertEqual(
+                result.output,
+                "google/pubsub/v1/pubsub.proto L160: Field behavior of an existing field `name` is changed.\n"
+                + "google/pubsub/v1/pubsub.proto L221: Field behavior of an existing field `topic` is changed.\n"
+                + "google/pubsub/v1/pubsub.proto L245: Field behavior of an existing field `topic` is changed.\n"
+                + "google/pubsub/v1/pubsub.proto L250: Field behavior of an existing field `messages` is changed.\n"
+                + "google/pubsub/v1/pubsub.proto L266: Field behavior of an existing field `project` is changed.\n"
+                + "google/pubsub/v1/pubsub.proto L267: A resource reference option is added to the field `project`, but it is not defined anywhere\n"
```
Comment (xiaozhenliu-gg5) [inline test/cli/test_detect.py:213]: The resource ref `"cloudresourcemanager.googleapis.com/Project"` is not defined in api-common-protos or API definition files, it is in `google3/google/cloudresourcemanager/v1/projects.proto` which is not in third_party folder so we do not have a way to include it as a protoc folder. It is a known resource lost breaking change. Same as below.

**Review by arithmetic1728 — APPROVED** [review]

File: test/cli/test_detect.py (hunk 1 above)
Comment (chrisdunelm) [reply test/cli/test_detect.py:213]: This resource is defined here:
https://github.com/googleapis/googleapis/blob/master/google/cloud/common_resources.proto
So this proto file should be included when running the detector for cloud APIs.

**Comment by chrisdunelm** [comment]
Please see if the common Project resource can be included, and therefore the expected errors reduced to zero.

File: test/cli/test_detect.py (hunk 1 above)
Comment (xiaozhenliu-gg5) [reply test/cli/test_detect.py]: I see, adding the `common_resources.proto` helps remove the errors! Thank you!

**Pull request merged**